src/index.js

```diff
@@ -12,18 +20,76 @@ module.exports = function () {
         startTime:      null,
         startTimes:     new Map(),
 
-        sauceTestReport: SauceJsonReporter.newReporter(),
+        // TestCafe Hooks
+        reportTaskStart: async function(startTime, userAgents, testCount, testStructure, properties) {            
+            this.sauceJsonReporter.reportTaskStart(startTime, userAgents, testCount);
+
+            if (this.disableSauceUpload) {
```

Comment:
Is there any other upload than to sauce?
```suggestion
            if (this.disableSauceUpload) {
```